### PR TITLE
-Add default to interface. Add fix for the test branch

### PR DESCRIPTION
**Pull request by amamounelsayed**

### CODE REVIEW

File: src/main/java/com/microsoft/azure/functions/ExecutionContext.java [hunk 1]
```diff
@@ -39,5 +39,7 @@
      * Returns the trace context.
      * @return the trace context
      */
-	TraceContext getTraceContext();
+    default TraceContext getTraceContext(){
+        return null;
```
Comment (pragnagopa) [inline src/main/java/com/microsoft/azure/functions/ExecutionContext.java]: instead of null, can we return empty strings?  @yojagad  thoughts?

Comment (yojagad) [reply src/main/java/com/microsoft/azure/functions/ExecutionContext.java]: Changed this to return a default TraceContext with empty strings and empty collection

Comment (amamounelsayed) [reply src/main/java/com/microsoft/azure/functions/ExecutionContext.java]: @pragnagopa  If we will return an object of the interface we will need to default all the functions that we have in TraceContext interface.

Comment (pragnagopa) [reply src/main/java/com/microsoft/azure/functions/ExecutionContext.java]: returning null from here will throw null ref if getTraceContext().getTraceparent() is called. I would prefer returning null/empty string which is what host would pass if there is no tracecontext being set.

**Pull request merged**